woocommerce/woocommerce
# [Experimental] Product Filters Redesign: Hide individual filter blocks from the Block Inserter

### Changes proposed in this Pull Request:







Closes #47525.



### How to test the changes in this Pull Request:

1. Navigate to Single Product template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "filter" in the block inserter.
5. Make sure you CANNOT see all these inner blocks: Product Filters: Active (Beta), Product Filters: Price (Beta), Product Filters: Stock Status (Beta), Product Filters: Attribute (Beta), Product Filters: Rating (Beta).



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
Hide individual filter blocks from the block inserter

</details>
